src/main/java/de/tum/cit/aet/artemis/core/web/open/PublicUserJwtResource.java

```diff
@@ -86,7 +86,7 @@ public ResponseEntity<Void> authorize(@Valid @RequestBody LoginVM loginVM, @Requ
             ResponseCookie responseCookie = jwtCookieService.buildLoginCookie(rememberMe);
             response.addHeader(HttpHeaders.SET_COOKIE, responseCookie.toString());
 
-            return ResponseEntity.ok().build();
+            return ResponseEntity.ok(responseCookie.getValue());
```

Comment:
Are we always returning the token regardless of whether it's been requested? I see no problem with it, but would like to discuss whether it makes sense to introduce an optional (default false) parameter of whether you want to receive the token :)

Reply:
yes we are always returning the token, as it does not make a difference security wise. Therefore its more consistent to have on route and the client can decide what to use

Reply:
But we got the `re-key` endpoint anyways, right? So why would we return the token at all in this method?

Reply:
that has nothing per se with the VSCode plugin to do. its just good to allow for an API cookies as well as bearer.

But we do need it for the Plugin as well, for the local machine use case where the user signs in with his credentials

Reply:
True that 👍